libs/live-config/package.json

```diff
@@ -27,6 +27,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "mergician": "^2.0.0"
```

Comment:
WDYT about using lodash instead ? It is already in the codebase and is 20 000x more downloaded per week

https://www.npmjs.com/package/lodash
https://www.npmjs.com/package/mergician